tests/tests/plugin/unit/test_trainable_tagger.py

```diff
@@ -79,7 +80,7 @@ def test_trainable_tagger():
             plugin_instance_id="000",
         ).dict()
     )
-    assert tagger2.data == TRAIN_RESPONSE.dict()
+    assert tagger2.data == {to_camel(k): v for (k, v) in TRAIN_RESPONSE.dict().items()}
```

Comment:
@EniasCailliau  Ditto; see above comment

Reply:
We need to enable use_alias while calling dict():

TRAIN_RESPONSE.dict(use_alias=True)

Reply:
Note: it turns out that it's `by_alias` -- but this comment helped me quickly find that. Thanks!